workflows/cloudify_system_workflows/snapshot.py

```diff
@@ -385,12 +385,61 @@ def _update_es_node(es_node):
             source['main_file_name'] = ''
 
 
+def _assert_clean_elasticsearch(es):
+    exception = NonRecoverableError('Manager is not clean')
```

Comment:
Do you mean something like:
def raise(msg):
    raise NonRecoverableError(msg)
?